### PR TITLE
JAMES-3443 Fetch textBody for parts nested inside multipart/alternative

### PR DESCRIPTION
**Pull request by LanKhuat**
https://github.com/linagora/james-project/issues/4028

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
test this please

File: server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/resources/eml/related_in_alternative_multipart.eml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Subject: Event <AT> Verify User Send Email To Attendees Tue November 10
+ 09:52:12 UTC 2020 from User A canceled
+Message-ID: <9c132e65-68c6-431d-e908-40cd76202019@qa.open-paas.org>
+X-Mailer: nodemailer (2.6.0; +http://nodemailer.com/;
+ SMTP/2.7.2[client:2.12.0])
+Date: Tue, 10 Nov 2020 02:57:03 +0000
+X-Smtpcorp-Track: 1kcJqCDIIfR_7f.qrGUrhUzsE74L
+Feedback-ID: 266739m:266739aja3LFS:266739spJBJJEdpb
+X-Report-Abuse: Please forward a copy of this message, including all headers,
+ to <abuse-report@smtp2go.com>
```
Comment (chibenwa) [inline server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/resources/eml/related_in_alternative_multipart.eml:59]: Please anonimise better the header of that email (ips, uneeded header fields, etc...)

**Review by chibenwa — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**